sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -272,6 +327,54 @@
     </item>
 </SALTelemetry>
 
+<SALTelemetry>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_balanceForceData</EFDB_Topic>
+    <Description>Net forces and moments as commanded by the force balance system. This is a component of the total force, which has the topics of netForcesTotal and netMomentsTotal.</Description>
```

Comment:
suggest removing `which has the topics of netForcesTotal and netMomentsTotal`. It just adds confusion.

Reply:
Maybe you could suggest a good description for this. I felt if I do not add this, it confused me as well. I do not understand this topic when I read the LTS-162. This is why I slacked you in the last week to understand it.

Reply:
Net forces and moments as commanded by the force balance system. These are calculated using the 72 axial and 6 tangent balance forces, which are a component of the total forces being commanded.

Reply:
Updated in commit: 7f353f2. Thanks!